src/index.js

```diff
@@ -56,7 +56,18 @@ module.exports = {
     'sort-vars': ['error'], // https://eslint.org/docs/rules/sort-vars
 
```

Comment:
Suggestion: always put the react imports at the top, not sorted. for consistency
```suggestion
'import/order': [
      'error',
      {
        pathGroups: [
          {
            pattern: 'react',
            group: 'builtin',
            position: 'before',
          },
        ],
        pathGroupsExcludedImportTypes: ['react'],
      },
    ],
```

Reply:
That doesn't make much sense to me.

I understand why you suggest it ("conditioned by the always-present react import pre v17"), but there's no logical reason to not adhere react to the same "alphabetic import" as other packages.

Reply:
I'm in the default order camp (no React exception).

Related: [simple-import-sort](https://www.npmjs.com/package/eslint-plugin-simple-import-sort) can fix more sorting warnings than `import/order`, will make PR

Reply:
in my eyes this is a legacy fix so older projects don't spam us with 1000 unneeded errors. new projects will not need this

Reply:
@thgh @JaccoGoris 

2 to 1 don't make exception for react?